server/paymentProviders/stripe/creditcard.js

```diff
@@ -308,11 +313,124 @@ export const setupCreditCard = async (paymentMethod, { user, collective } = {})
   return paymentMethod;
 };
 
+// Charge has a new Stripe dispute
+const createDispute = async event => {
+  const stripeChargeId = event.data.object.charge;
+  const transactions = await models.Transaction.findAll({
+    where: { data: { charge: { id: stripeChargeId } } },
```

Comment:
I think we should create an index either for the json column `data` or specifically for this `charge.id` field,
the transaction is our biggest table and I think this select will cause a slow table scan.

```
select id from "Transactions" t where t.data->'charge'->>'id' = 'ch_3L4DagBYycQg1OMf1KxilamS'

Gather  (cost=1000.00..181144.54 rows=16100 width=395)                                           
  Workers Planned: 1                                                                             
  ->  Parallel Seq Scan on "Transactions" t  (cost=0.00..178534.54 rows=9471 width=395)          
        Filter: (((data -> 'charge'::text) ->> 'id'::text) = 'ch_3L4DagBYycQg1OMf1KxilamS'::text)

2 row(s) fetched - 46.702s
```